src/containers/Header/UserInfo/index.jsx

```diff
@@ -144,10 +144,10 @@ class UserInfo extends PureComponent {
   _renderUser() {
     const classes = classnames(styles.UserInfo_container_user_img, testClass('user-picture'));
     return <AdultPicture className={classes}
-        src={this.props.user.profile.avatar_url}
-        gender={this.props.user.profile.gender}
-        userFirstName={this.props.user.profile.given_name}
-        userLastName={this.props.user.profile.family_name}
+        src={this.props.user.get('profile').avatar_url}
+        gender={this.props.user.get('profile').gender}
+        userFirstName={this.props.user.get('profile').given_name}
+        userLastName={this.props.user.get('profile').family_name}
```

Comment:
I've been thinking that one of our mistakes was to use combinations of `Map`s and `Record`s that make it really annoying to know whether you can call a property directly or if you need to use `get()`. I think a better approach we could turn to is to always use `Map`s and `get()`.